### PR TITLE
This PR adds the class FrankaEmikaPandaRobot()

### PR DESCRIPTION
**Pull request by ffasilva**
# Main instructions

By submitting this pull request, you automatically agree that you have read and accepted the following conditions:

- Anyone wanting to propose a new modification or introduce new functionality should reach out to the team first, as proposed modifications that do not comply with the library's development philosophy and style, do not follow the library's architecture, do not introduce a clear and general benefit to the library other than to the person who proposed the modification will likely be rejected with no further discussion.
- Support for DQ Robotics is given voluntarily and it is not the developers' role to educate and/or convince anyone of their vision.
- Any possible response and its timeliness will be based on the relevance, accuracy, and politeness of a request and the following discussion.
- You are familiar with the [development workflow](https://github.com/dqrobotics/matlab/blob/master/CONTRIBUTING.md).
- Each pull request should contain only individual changes (several changes of the same type **are allowed** on the same pull request).
- Refactoring or modifying internal implementation that is working is not allowed unless comprehensively discussed with and approved by @bvadorno and @mmmarinho.

## Description of changes

Hi @dqrobotics/developers,

This PR adds the class `FrankaEmikaPandaRobot`. An example of application is given in [7](https://github.com/dqrobotics/matlab-examples/pull/7).

Kind regards,
Frederico

## Rationale

The Franka Emika Panda is a commonly used robot manipulator, but the library currently does not provide its class.

P.S.: I had to redo this PR because I made the mistake of initiating it from my `master` the first time.

### CODE REVIEW

File: robots/FrankaEmikaPandaRobot.m [hunk 1]
```diff
@@ -0,0 +1,55 @@
+% panda = FrankaEmikaPandaRobot.kinematics() returns a DQ_kinematics object
+% using the modified Denavit-Hartenberg parameters of the Franka Emika
+% Panda robot
+
+% (C) Copyright 2015-2022 DQ Robotics Developers
```
Comment (bvadorno) [inline robots/FrankaEmikaPandaRobot.m:5]: The date range is incorrect. It should be 2011-2023

File: robots/FrankaEmikaPandaRobot.m [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+% DQ Robotics website: dqrobotics.github.io
+%
+% Contributors to this file:
+%     Frederico Fernandes Afonso Silva - frederico.silva@ieee.org
+
+classdef FrankaEmikaPandaRobot
+    methods (Static)
+        function ret = kinematics()
+            % Modified D-H of Franka Emika Panda
+            FEp_DH_theta = [0, 0, 0, 0, 0, 0, 0];
```
Comment (bvadorno) [inline robots/FrankaEmikaPandaRobot.m:31]: Let's make things simple. Change FEp_DH_theta to mDH_theta. Please, do the same for all other variables.

File: robots/FrankaEmikaPandaRobot.m [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+            FEp_DH_d = [0.333, 0, 3.16e-1, 0, 3.84e-1, 0, 0];
+            FEp_DH_a = [0, 0, 0, 8.25e-2, -8.25e-2, 0, 8.8e-2];
+	        FEp_DH_alpha = [0, -pi/2, pi/2, pi/2, -pi/2, pi/2, pi/2];
```
Comment (bvadorno) [inline robots/FrankaEmikaPandaRobot.m:34]: Remove indentation.

File: robots/FrankaEmikaPandaRobot.m [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            FEp_DH_type = repmat(DQ_SerialManipulatorDH.JOINT_ROTATIONAL,1,7);
+
+	        FEp_DH_matrix = [FEp_DH_theta;
```
Comment (bvadorno) [inline robots/FrankaEmikaPandaRobot.m:37]: Please adjust the indentation.

**Review by bvadorno — CHANGES_REQUESTED** [review]

File: robots/FrankaEmikaPandaRobot.m (hunk 1 above)
Comment (ffasilva) [reply robots/FrankaEmikaPandaRobot.m]: Fixed in commit [81fe84a](https://github.com/dqrobotics/matlab/pull/102/commits/81fe84a53a0139e36040c25f8f38e4893855bd16).

File: robots/FrankaEmikaPandaRobot.m (hunk 2 above)
Comment (ffasilva) [reply robots/FrankaEmikaPandaRobot.m]: Fixed in commit [8ce7d82](https://github.com/ffasilva/matlab/commit/8ce7d82d06244e07cbd5012fd2623e4b32973a3d).

File: robots/FrankaEmikaPandaRobot.m (hunk 3 above)
Comment (ffasilva) [reply robots/FrankaEmikaPandaRobot.m]: Fixed in commit [3a862b7](https://github.com/ffasilva/matlab/commit/3a862b77294dd1b268bbc09dcf75e35a4cb6d012).

File: robots/FrankaEmikaPandaRobot.m (hunk 4 above)
Comment (ffasilva) [reply robots/FrankaEmikaPandaRobot.m]: Fixed in commit [21a2174](https://github.com/ffasilva/matlab/commit/21a2174bb008cb2fa9cf690f6a1206d1f54126a6).

**Review by bvadorno — APPROVED** [review]

**Pull request merged**